journals/apps/journals/management/commands/publish_journals.py

```diff
@@ -0,0 +1,168 @@
+'''management command to publish journals'''
+from __future__ import unicode_literals
+
+from django.core.management.base import BaseCommand, CommandError
+from hashlib import md5
+from journals.apps.journals.models import Journal, JournalIndexPage, JournalAboutPage, JournalMetaData
+from slumber.exceptions import HttpClientError
+from wagtail.wagtailcore.models import Page
+
+
+class Command(BaseCommand):
+    '''Base class for management command'''
+    help = 'Publish Journal information to discovery and ecommerce'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--create', help='The journal name')
+        parser.add_argument('--org', help='Organization name (from journals-service)')
+        parser.add_argument('--price', help='Journal price')
+        parser.add_argument('--currency', default='USD', help='Journal currency')
+        parser.add_argument('--access_length', type=int, default=365,
+                            help='number of days to allow access')
+        parser.add_argument('--delete', help='The journal uuid to delete')
+        parser.add_argument('--publish', dest='publish', action='store_true',
+                            help='Create the journal live and publish it live on site')
+        parser.add_argument('--no-publish', dest='publish', action='store_false',
+                            help='Create the journal but do not make it live on site')
+        parser.set_defaults(publish=True)
+
+    def _handle_create(self, options):
+        '''handle creation of new Journal'''
+        name = options['create']
+        org = options['org']
+        price = options['price']
+        currency = options['currency']
+        access_length = options['access_length']
+        publish = options['publish']
+
+        if not org:
+            raise CommandError('--org <Org Name> must be specified')
+        if not price:
+            raise CommandError('--price <Journal price> must be specified')
+
+        try:
+            journal = Journal.objects.create_journal(name, org, access_length)
+            journal_about_page = self._update_wagtail_pages(journal, create=True if name else False, publish=publish)
```

Comment:
Don't know why Github hid the comment, so I'm positing it again:
`create` will always be `True` since `_handle_create()` is only called if `options['create']`is truthy.